app/controllers/school_suggestions_controller.rb

```diff
@@ -0,0 +1,11 @@
+class SchoolSuggestionsController < ApplicationController
+  def index
+    render json: GiasSchool.search_name_urn_postcode(query_params)
+  end
+
+  private
+
+  def query_params
+    params.require(:query)&.downcase
+  end
+end
```

Comment:
I'm wondering, does this deserve to go into an API controller instead? `Api::SchoolsController` maybe? That way we can do `GET /api/schools?query=abc`.

cc: @JamieCleare2525

Reply:
Sure, it is an api call after all

Reply:
That convention is a bit weird to me, as I have always worked from the point of view that API controllers deal with APIs we have written to be consumed by other applications. Not for the retrieval of information from other APIs. However, I'm happy if we feel this will work for us.

Reply:
I see it as a separation of response expectations. `/api/** returns `json` responses and `!/app/**` returns `html`. It would be an internal API, for external APIs, I'd recommend versioning the API, but we don't have to go there.